### ISSUE TITLE
paket pack with CLI CPP projects

### ISSUE THREAD

**Issue by pavelhritonenko** [fsprojects/Paket#1870]
### Description

I can't create nuget package from CLI CPP library

### Repro steps

I have `./.paket/` in my `PATH` environment variable

1. Create empty CPP project (I used VS2015)
1. run `paket init` (I actually run `../other-project/.paket/paket init`)
1. run `paket.bootstrapper`
1. run `paket install`
1. add `paket.template` with content `type project`
1. run `paket pack output .`

### Expected behavior

Paket creates "empty" nuget package

### Actual behavior

```$ paket pack output .
Paket version 3.15.0.0
Paket failed with:
        Unable to find Release output path node in file C:\projects\paket-cpp\ClassLibrary1\ClassLibrary1\ClassLibrary1.vcxproj for any known platforms```

### Related information 

* Latest Windows 10 (Red stone)
* .NET 4.6.1

### Attached project (without binaries or paket.exe)
[paket-cpp.zip](https://github.com/fsprojects/Paket/files/432259/paket-cpp.zip)

**Comment by forki** [comment]
how would you detect the output path in that file?

**Comment by pavelhritonenko** [comment]
I changed outdir in project options and now I have this change in .vcxproj:

![image](https://cloud.githubusercontent.com/assets/2510026/17890029/12aa484a-693c-11e6-8610-e397f96d1661.png)

By default it is `$(SolutionDir)$(Platform)\$(Configuration)\`

**Comment by forki** [comment]
So after the changes it pack works?

Am 23.08.2016 1:16 nachm. schrieb "pavelhritonenko" <
notifications@github.com>:

**Comment by pavelhritonenko** [comment]
No, it doesn't (even if I specified `OutDir` node for all possible platforms)

**Comment by forki** [comment]
I'm pushing a fix which would also check for OutDir (currently we look for OutputPath), but I guess we can't use the default. Not sure how we would do this

**Comment by forki** [comment]
can you please retry?

**Comment by pavelhritonenko** [comment]
First, thanks for instant reaction, it's amazing.

Next error was related to absent `<AssemblyName />` tag in the .vcxproj file, I've added it manually and now I get this when run `paket pack output . buildplatform x64`:

```
$ paket pack output . buildplatform x64
Paket version 3.15.1.0
Paket failed with:
        The input sequence was empty.
Parameter name: source
```

**Comment by forki** [comment]
can you please upload zip again?

**Comment by forki** [comment]
the newly changed one I mean

**Comment by pavelhritonenko** [comment]
https://github.com/pavelhritonenko/paket-cpp - I noticed there is unnecessary level of folders however I decided to not touch it.

**Comment by forki** [comment]
and another release. please give it a try

**Comment by pavelhritonenko** [comment]
Version from repository I linked:
```
$ paket pack output . buildplatform x64
Paket version 3.15.2.0
Paket failed with:
        Index was outside the bounds of the array.
```

**Comment by forki** [comment]
Since I can't build your project. Please commit output binaries as well

Am 23.08.2016 17:45 schrieb "pavelhritonenko" <notifications@github.com>:

**Comment by forki** [comment]
so that a clean checkout would produce that error

**Comment by pavelhritonenko** [comment]
I commited binaries. Did you mean that?

```
Khritonenko@khritonenko MINGW64 /C/projects/paket-cpp (master)
$ git clean -fXd
Removing .paket/paket.exe

Khritonenko@khritonenko MINGW64 /C/projects/paket-cpp (master)
$ git clean -fxd

Khritonenko@khritonenko MINGW64 /C/projects/paket-cpp (master)
$ paket.bootstrapper.exe
Checking Paket version (downloading latest stable)...
Copying version 3.15.2 from cache.
Done.

Khritonenko@khritonenko MINGW64 /C/projects/paket-cpp (master)
$ paket pack output . buildplatform x64
Paket version 3.15.2.0
Paket failed with:
        Index was outside the bounds of the array.
```

**Comment by forki** [comment]
sorry for playing pingpong. with latest version (just released) I now see:

![image](https://cloud.githubusercontent.com/assets/57396/17905845/f35d9314-6974-11e6-85c6-b44771f38560.png)

this seems to be correct, right?

**Comment by forki** [comment]
"Missing: Authors"
